PDO/PDO.php

```diff
@@ -710,6 +710,18 @@ class PDO  {
 	 */
 	const MYSQL_ATTR_MULTI_STATEMENTS = 1015;
 
+	/**
+	 * <p>
+	 * Disables SSL peer verification when set to FALSE.
+	 * </p>
+	 * <p>
+	 * Note, this constant is not yet documented on the php.net documentation.
```

Comment:
I think, this note is redundant and it's better to remove it

Reply:
Okay, i thought that it was a good idea for those that have to maintain the project.

Reply:
@andrey-sokolov It's fixed !

Reply:
@andrey-sokolov up :)

Reply:
@andrey-sokolov Up !